app/views/templates/ssh/register_content_host.erb

```diff
@@ -1,32 +1,38 @@
 <%#
-kind: job_template
 name: Register Content Host - SSH Default
 job_category: Commands
-description_format: "Register Content Host using activation key %{activation key}"
-provider_type: SSH
+description_format: Register Content Host using activation key %{activation key}
+snippet: false
 template_inputs:
 - name: activation key
-  description: Name of the Activation Key to use for registering the host
-  input_type: user
   required: true
-- name: organization name
-  description: Name of the Organization to register the host to
   input_type: user
+  description: Name of the Activation Key to use for registering the host
+  advanced: false
+- name: organization name
   required: true
+  input_type: user
+  description: Name of the Organization to register the host to
+  advanced: false
+provider_type: SSH
+kind: job_template
 %>
+
 # Install Katello's certificates
-yum localinstall -y <%= foreman_server_url %>/pub/katello-ca-consumer-latest.noarch.rpm
+curl -k -L -O "<%= foreman_server_url %>/pub/katello-ca-consumer-latest.noarch.rpm"
+yum localinstall -y ./katello-ca-consumer-latest.noarch.rpm
```

Comment:
yum was refusing to install a package from an URL, might be some kind of misconfiguration on my side